source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidSavedGameClient.cs

```diff
@@ -98,38 +99,37 @@ public AndroidSavedGameClient(AndroidJavaObject account)
         {
             int conflictPolicy; // SnapshotsClient.java#RetentionPolicy
-            switch (resolutionStrategy) {
-            case ConflictResolutionStrategy.UseLastKnownGood:
-                conflictPolicy = 2 /* RESOLUTION_POLICY_LAST_KNOWN_GOOD */;
-                break;
-            case ConflictResolutionStrategy.UseMostRecentlySaved:
-                conflictPolicy = 3 /* RESOLUTION_POLICY_MOST_RECENTLY_MODIFIED */;
-                break;
-            case ConflictResolutionStrategy.UseLongestPlaytime:
-                conflictPolicy = 1 /* RESOLUTION_POLICY_LONGEST_PLAYTIME*/;
-                break;
-            case ConflictResolutionStrategy.UseManual:
-                conflictPolicy = -1 /* RESOLUTION_POLICY_MANUAL */;
-                break;
-            default:
-                conflictPolicy = 3 /* RESOLUTION_POLICY_MOST_RECENTLY_MODIFIED */;
-                break;
+            switch (resolutionStrategy)
+            {
+                case ConflictResolutionStrategy.UseLastKnownGood:
+                    conflictPolicy = 2 /* RESOLUTION_POLICY_LAST_KNOWN_GOOD */;
+                    break;
+                case ConflictResolutionStrategy.UseMostRecentlySaved:
+                    conflictPolicy = 3 /* RESOLUTION_POLICY_MOST_RECENTLY_MODIFIED */;
+                    break;
+                case ConflictResolutionStrategy.UseLongestPlaytime:
+                    conflictPolicy = 1 /* RESOLUTION_POLICY_LONGEST_PLAYTIME*/;
+                    break;
+                case ConflictResolutionStrategy.UseManual:
+                    conflictPolicy = -1 /* RESOLUTION_POLICY_MANUAL */;
+                    break;
+                default:
+                    conflictPolicy = 3 /* RESOLUTION_POLICY_MOST_RECENTLY_MODIFIED */;
+                    break;
             }
 
-            using (var task = mSnapshotsClient.Call<AndroidJavaObject>("open", filename, /* createIfNotFound= */true, conflictPolicy))
-            {   // Task<DataOrConflict<Snapshot>> task
-                task.Call<AndroidJavaObject>("addOnSuccessListener", new TaskOnSuccessProxy<AndroidJavaObject>(
+            using (var task = mSnapshotsClient.Call<AndroidJavaObject>("open", filename, /* createIfNotFound= */ true, conflictPolicy))
+            {
+                AndroidTaskUtils.AddOnSuccessListener<AndroidJavaObject>(
+                    task,
                     dataOrConflict => {
                         if (dataOrConflict.Call<bool>("isConflict"))
                         {
                             using (var conflict = dataOrConflict.Call<AndroidJavaObject>("getConflict"))
                             {
-                                // If we get here, manual conflict resolution is required.
                                 AndroidSnapshotMetadata original = new AndroidSnapshotMetadata(conflict.Call<AndroidJavaObject>("getSnapshot"));
```

Comment:
using for conflict.Call<AndroidJavaObject>("getSnapshot") and for conflict.Call<AndroidJavaObject>("getConflictingSnapshot")